src/app/containers/ArticleTimestamp/shouldDisplayLastUpdatedTimestamp.test.jsx

```diff
@@ -0,0 +1,132 @@
+import shouldDisplayLastUpdatedTimestamp, {
+  hasBeenUpdated,
+  publishedAndUpdatedToday,
+} from './shouldDisplayLastUpdatedTimestamp';
+import { timestampGenerator, sameDayTimestampsGenerator } from './testHelpers';
+
+describe('shouldDisplayLastUpdatedTimestamp functions', () => {
+  let mockedDate;
```

Comment:
I think this can now be deleted since you're using `jest.resetAllMocks();`

Reply:
I can yea I just thought it was nicer to make it clear that we're using a mocked date, referring to mockedDate throughout, rather than Date.now and it potentially being assumed that it is indeed 'now'.

Reply:
Oh yes I see what you mean - makes sense to leave it as is.